zipline/finance/performance/period.py

```diff
@@ -87,6 +87,28 @@
 log = logbook.Logger('Performance')
 
 
+class FastSeries(object):
```

Comment:
Should we rename this, when I see `FastSeries`, I think it is an object that is like a series but faster; however, this is actually write-only and does not behave much like a series.